src/Commands/AdminCommands/CleanupCommand.php

```diff
@@ -327,7 +351,7 @@ private function getQueries($settings)
     public function execute()
     {
         $message = $this->getMessage();
-        $chat_id = $message->getFrom()->getId();
+        $chat_id = $message->getChat()->getId();
```

Comment:
Agreed, it's a rare case, but it can happen by mistake. Using `getFrom()` is a fail-safe for that, as the Chat ID would be the same as the From ID anyway in a private chat, so it doesn't make a difference there.

Haven't tested this yet, but can a bot delete a message from a user when it's admin in a group/channel? Then, we could even go as far as to offer the option of deleting a "by mistake" command execution.

What do you think?